qualtran/serialization/bloq_test.py

```diff
@@ -124,13 +124,6 @@ def test_cbloq_to_proto_test_mod_exp():
     num_binst = len(set(binst.bloq for binst in cbloq.bloq_instances))
     assert len(proto_lib.table) == 1 + num_binst
 
-    cbloq = ControlledBloq(mod_exp)
-    proto_lib = bloq_serialization.bloqs_to_proto(cbloq, max_depth=1)
-    # 2x that of ModExp.make_for_shor(17 * 19).decompose_bloq() because each bloq in the
-    # decomposition is now controlled and each Controlled(subbloq) requires 2 entries in the
-    # table - one for ControlledBloq and second for subbloq.
-    assert len(proto_lib.table) == 2 * (1 + num_binst)
-
```

Comment:
TODO(Tanuj, Arren): Add this test back once we update serialization code to support newly added `CtrlSpec` and corresponding `ControlledBloq`.

Reply:
https://github.com/quantumlib/Qualtran/issues/648